eel/tests/p2p_connection_test.rs

```diff
@@ -31,23 +33,26 @@ mod p2p_connection_test {
     #[file_serial(key, "/tmp/3l-int-tests-lock")]
     fn test_p2p_connection_with_unreliable_lsp() {
         nigiri::ensure_environment_running();
+
+        // Start the node when lspd isn't available
+        nigiri::pause_lspd();
         let node = mocked_storage_node().start_or_panic();
 
-        // Test disconnect when LSP is down.
-        {
-            // Let's shutdown LSPD LND.
-            nigiri::pause_lspd();
-            wait_for_eq!(node.get_node_info().num_peers, 0);
-        }
+        assert_eq!(
+            node.query_lsp_fee(),
+            Err(runtime_error(
+                RuntimeErrorCode::LspServiceUnavailable,
+                "Failed to get LSP info"
+            ))
+        );
+        wait_for_eq!(node.get_node_info().num_peers, 0);
```

Comment:
Would it be possible to move this code block in front of `nigiri::ensure_environment_running();` such that `lspd` does not need pausing? Or is there something else required from the environment such that the node couldn't be started?

Reply:
Not sure I understand what you propose. The goal is to start the node without having lspd running. If we start the node before `nigiri::ensure_environment_running();`, we don't know if lspd is running or not.

Reply:
Fixed: https://github.com/getlipa/lipa-lightning-lib/compare/40c9a8aed3598b56cd2277cb3c6f7e04e819e93d..92a9f39937a2a6e383be57c891e6a3ab6b274fc3